### PR TITLE
Rename `RequestedSize` to `WindowSize` and `RequestedPosition` to `WindowPosition`

### PR DESCRIPTION
**Pull request by tronical**
…ndowPosition`

### CODE REVIEW

**Review by hunger — CHANGES_REQUESTED** [review]
I think you missed one occurance in internal/backends/testing/lib.rs

File: internal/core/api.rs [hunk 1]
```diff
@@ -81,21 +81,21 @@ impl PhysicalPosition {
     }
 }
 
-/// The requested position to render the window at.
+/// The position of the window in either physical or logical pixels.
```
Comment (ogoffart) [inline internal/core/api.rs:84]: unrelated to this path, but i think it would be nice to reference ``[`Window::set_position`]``

**Comment by tronical** [comment]
> I think you missed one occurance in internal/backends/testing/lib.rs

Where? set_size isn't implemented there, and set_position is changed.

File: internal/core/api.rs (hunk 1 above)
Comment (tronical) [reply internal/core/api.rs:84]: Good idea, let me try that

**Comment by hunger** [comment]
@tronical: Ripgrep found this:

```
internal/backends/testing/lib.rs
62:    fn set_position(&self, _position: i_slint_core::api::RequestedPosition) {
```

**Review by hunger — APPROVED** [review]

**Pull request merged**